main.go

```diff
@@ -26,6 +28,7 @@ var (
 	flagCount = flag.Int("n", 15, "iterations")
 	flag386   = flag.Bool("386", false, "run in 386 mode")
 	flagEach  = flag.Bool("each", false, "run for every commit between before and after")
+	flagCl    = flag.String("cl", "", "run benchmark on CL number")
```

Comment:
flag.CL please. Maybe flag.Int instead of flag.String?

Reply:
Fixed